blockchain/source/contracts/DeployList.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.23;
+
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+
+contract DeployList is Ownable {
+    address[]deployers;
+
+    event DeployerAdded(address deployer);
+
+    event DeployerRemoved(address deployer);
+
+    constructor(address[] _deployers){
+        owner = msg.sender;
+        deployers = _deployers;
+    }
+
+    function addDeployer(address _deployer) public onlyOwner {
+        deployers.push(_deployer);
+        DeployerAdded(_deployer);
+    }
+
+    function removeDeployer(address _deployer) public onlyOwner {
+        for (uint i = 0; i < deployers.length - 1; i++)
+            if (deployers[i] == _deployer) {
+                deployers[i] = deployers[deployers.length - 1];
+                break;
+            }
+        deployers.length -= 1;
```

Comment:
It completely weird to change array length by hand, why not to use a mapping here?

Reply:
fuuuuuuuuuuuuuuuuu